frontend/src/Activity/Blocklist/BlocklistConnector.js

```diff
@@ -4,116 +4,116 @@ import { connect } from 'react-redux';
+        isClearingBlocklistExecuting,
+        ...blocklist
       };
     }
   );
 }
 
 const mapDispatchToProps = {
-  ...blacklistActions,
+  ...blocklistActions,
   executeCommand
 };
 
-class BlacklistConnector extends Component {
+class BlocklistConnector extends Component {
 
   //
   // Lifecycle
 
   componentDidMount() {
     const {
       useCurrentPage,
-      fetchBlacklist,
-      gotoBlacklistFirstPage
+      fetchBlocklist,
+      gotoBlocklistFirstPage
     } = this.props;
 
     registerPagePopulator(this.repopulate);
 
     if (useCurrentPage) {
-      fetchBlacklist();
+      fetchBlocklist();
     } else {
-      gotoBlacklistFirstPage();
+      gotoBlocklistFirstPage();
     }
   }
 
   componentDidUpdate(prevProps) {
-    if (prevProps.isClearingBlacklistExecuting && !this.props.isClearingBlacklistExecuting) {
-      this.props.gotoBlacklistFirstPage();
+    if (prevProps.isClearingBlocklistExecuting && !this.props.isClearingBlocklistExecuting) {
+      this.props.gotoBlocklistFirstPage();
     }
   }
 
   componentWillUnmount() {
-    this.props.clearBlacklist();
+    this.props.clearBlocklist();
     unregisterPagePopulator(this.repopulate);
   }
 
   //
   // Control
 
   repopulate = () => {
-    this.props.fetchBlacklist();
+    this.props.fetchBlocklist();
   }
   //
   // Listeners
 
   onFirstPagePress = () => {
-    this.props.gotoBlacklistFirstPage();
+    this.props.gotoBlocklistFirstPage();
   }
 
   onPreviousPagePress = () => {
-    this.props.gotoBlacklistPreviousPage();
+    this.props.gotoBlocklistPreviousPage();
   }
 
   onNextPagePress = () => {
-    this.props.gotoBlacklistNextPage();
+    this.props.gotoBlocklistNextPage();
   }
 
   onLastPagePress = () => {
-    this.props.gotoBlacklistLastPage();
+    this.props.gotoBlocklistLastPage();
   }
 
   onPageSelect = (page) => {
-    this.props.gotoBlacklistPage({ page });
+    this.props.gotoBlocklistPage({ page });
   }
 
   onRemoveSelected = (ids) => {
-    this.props.removeBlacklistItems({ ids });
+    this.props.removeBlocklistItems({ ids });
   }
 
   onSortPress = (sortKey) => {
-    this.props.setBlacklistSort({ sortKey });
+    this.props.setBlocklistSort({ sortKey });
   }
 
   onTableOptionChange = (payload) => {
-    this.props.setBlacklistTableOption(payload);
+    this.props.setBlocklistTableOption(payload);
 
     if (payload.pageSize) {
-      this.props.gotoBlacklistFirstPage();
+      this.props.gotoBlocklistFirstPage();
     }
   }
 
-  onClearBlacklistPress = () => {
-    this.props.executeCommand({ name: commandNames.CLEAR_BLACKLIST });
+  onClearBlocklistPress = () => {
+    this.props.executeCommand({ name: commandNames.CLEAR_BLOCKLIST });
   }
 
   onTableOptionChange = (payload) => {
-    this.props.setBlacklistTableOption(payload);
+    this.props.setBlocklistTableOption(payload);
 
     if (payload.pageSize) {
-      this.props.gotoBlacklistFirstPage();
+      this.props.gotoBlocklistFirstPage();
     }
   }
```

Comment:
Reminder to self to remove one of these (unnecessary existing duplication of function)